src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -22,6 +22,7 @@
 public class ParserUtil {
 
     public static final String MESSAGE_INVALID_INDEX = "Index is not a non-zero unsigned integer.";
+    private static final String NULL_STRING = "null_string";
```

Comment:
If this constant were to be used, it should be imported from one location instead of duplicating the instances, so that there is a single source of truth. What do you think?

Reply:
Also, what if for some odd reason a correctly named group name is actually "null_string"? That would cause issues too

Perhaps checking that getValue(PREFIX_GROUP) returns null/empty/nothing and then having parseGroup() return a empty Group() would be better?

Relevant code wrt 2nd sentence/paragraph ^
 `ParserUtil.parseGroup(argMultimap.getValue(PREFIX_GROUP).orElse(NULL_STRING));`

Reply:
THe problem with getValue returning a null is that since it requires a type of string, null will throw an error. I can change it to an empty string and make the changes with cached empty group and all though.

Reply:
yup, i think that'll be ideal!